src/screens/FundCampaignPledge/modal/PledgeModal.spec.tsx

```
@@ -266,4 +383,221 @@ describe('PledgeModal', () => {
+      expect(amountInput).toHaveAttribute('value', '0');
+      expect(screen.getByLabelText('Currency')).toBeInTheDocument();
+      const startDateInput = screen.getByLabelText('Start Date');
+      const endDateInput = screen.getByLabelText('End Date');
+      expect(startDateInput).toBeDisabled();
+      expect(endDateInput).toBeDisabled();
+    });
+  });
+
+  it('should handle missing pledgeUsers array', async () => {
+    const invalidPledge = {
+      ...pledgeProps[1].pledge!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)